kitsune/questions/config.py

```diff
@@ -639,6 +639,17 @@
                 ),
             },
         ),
+        (
+            "mozilla-account",
```

Comment:
This is already part of another PR. We should keep one out of the two.

Reply:
It will be different now, I had to revert to `firefox_accounts` for the product.